apps/gp2-frontend/package.json

```diff
@@ -14,41 +14,39 @@
     "release:frontend": "node ./scripts/sentry-release.js"
   },
   "dependencies": {
+    "@asap-hub/algolia": "workspace:*",
     "@asap-hub/auth": "workspace:*",
+    "@asap-hub/fixtures": "workspace:*",
     "@asap-hub/flags": "workspace:*",
     "@asap-hub/frontend-utils": "workspace:*",
     "@asap-hub/gp2-components": "workspace:*",
+    "@asap-hub/model": "workspace:*",
     "@asap-hub/react-components": "workspace:*",
     "@asap-hub/react-context": "workspace:*",
     "@asap-hub/routing": "workspace:*",
     "@asap-hub/unsupported-browser-page": "workspace:*",
     "@asap-hub/validation": "workspace:*",
     "@auth0/auth0-spa-js": "1.22.4",
-    "@fast-csv/format": "4.3.5",
     "@sentry/react": "6.11.0",
     "@sentry/tracing": "6.11.0",
     "browser-image-compression": "1.0.17",
     "date-fns": "2.29.3",
     "history": "4.10.1",
+    "path": "npm:path-browserify@1.0.1",
     "react": "17.0.2",
     "react-app-polyfill": "3.0.0",
     "react-dom": "17.0.2",
     "react-error-boundary": "3.1.4",
     "react-router-dom": "5.3.3",
     "react-router-last-location": "2.0.1",
     "recoil": "patch:recoil@^0.1.2#../../patches/recoil.patch",
-    "streamsaver": "2.0.6",
```

Comment:
There is not export functionality on GP2 ATM. If there is we should move it to a shared module or maybe into the algolia library.